flashlight/app/lm/CMakeLists.txt

```diff
@@ -1,52 +1,27 @@
 cmake_minimum_required(VERSION 3.10)
 
-add_library(
-  flashlight-app-lm
-  ""
-  )
-
-target_sources(
-  flashlight-app-lm
+# ----------------------------- Binaries -----------------------------
+add_executable(fl_lm_train
+  ${CMAKE_CURRENT_LIST_DIR}/Train.cpp
+)
+target_sources(fl_lm_train
   PRIVATE
   ${CMAKE_CURRENT_LIST_DIR}/Trainer.cpp
-  )
-
-target_link_libraries(
-  flashlight-app-lm
-  PUBLIC
-  ${GLOG_LIBRARIES}
-  ${gflags_LIBRARIES}
-  flashlight
-  fl-libraries
-  flashlight-app-common
-  )
-
-target_include_directories(
-  flashlight-app-lm
-  PUBLIC
-  ${GLOG_INCLUDE_DIRS}
-  ${gflags_INCLUDE_DIRS}
-  )
-
-# ------------------------ Components ------------------------
-
-# Common
-include(${CMAKE_CURRENT_LIST_DIR}/common/CMakeLists.txt)
-
-# Data
-include(${CMAKE_CURRENT_LIST_DIR}/data/CMakeLists.txt)
-
-# ----------------------------- Binaries -----------------------------
-add_executable(fl_lm_train ${CMAKE_CURRENT_LIST_DIR}/Train.cpp)
+)
```

Comment:
Eh, annoying that the LM trainer is still here/built in separate binaries. Note for later: ideal outcome would be:
1. Generic trainer that lives in the `runtime` pkg
2. Trainer with no glog/gflags dependency that lives in the LM pkg

For now, this is probably the best we can do

Reply:
Agree on both points. Current "Trainer" is really an LMTrainer though and will take more work to generalize. Outside of this scope for now.